// Copyright 2023 Intrinsic Innovation LLC

syntax = "proto3";

package intrinsic_proto.perception;

// Encapsulates pose estimator id.
// This message type allows the UI to interpret the field as a pose estimator
// and automatically hook it up to the available pose estimators.
message PoseEstimatorId {
  string id = 1;
}
